cpp/CMakePresets.json

```diff
@@ -6,6 +6,30 @@
     "patch": 0
   },
   "configurePresets": [
+    {
+      "name": "_allocator-none",
+      "hidden": true,
+      "cacheVariables": {
+        "ARROW_JEMALLOC": "OFF",
+        "ARROW_MIMALLOC": "OFF"
+      }
+    },
+    {
+      "name": "_allocator-jemalloc",
+      "hidden": true,
+      "cacheVariables": {
+        "ARROW_JEMALLOC": "ON",
+        "ARROW_MIMALLOC": "OFF"
+      }
+    },
+    {
+      "name": "_allocator-mimalloc",
```

Comment:
Can we use this in `features-basic`?

Reply:
As replied in my other comment, yes we can. Will try.

Reply:
Addressed.